sb3_contrib/common/utils.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+import torch as th
+
+
+def quantile_huber_loss(
+    current_quantiles: th.Tensor,
+    target_quantiles: th.Tensor,
+    cum_prob: Optional[th.Tensor] = None,
```

Comment:
those are also called "midpoints" in the paper, no?

Reply:
Yes in QR-DQN paper, but it's less common and not used in other papers (IQN, FQF).

Reply:
maybe just add a comment saying "also called midpoints in QR-DQN paper"